src/store/buoy.ts

```diff
@@ -93,8 +95,8 @@ const createStore = (config: BuoyConfig, bustCache = false) => {
 
       // just fetch data, don't save anything
       async fetchBuoyData({ ids, variables, start, end }): Promise<Dataset> {
-        const startDate = start || this.minDate;
-        const endDate = end || this.maxDate;
+        const startDate = start || this.minDateRaw;
+        const endDate = end || this.maxDateRaw;
         return await erddapGet(
           `/${route}/query?ids=${ids}&variables=${variables}&start=${startDate}&end=${endDate}`
         );
```

Comment:
Since you're passing endDate into the URL, i'd expect it to be in yyyy-mm-dd format. Is that the case here? Should the raw date be converted to localISODate here?

Reply:
we do have getters further down in the script that we can use and just pass `minDate` and `maxDate`